packages/components/src/components/Table/Table.stories.tsx

```diff
@@ -196,6 +218,21 @@ export const ReactTable = (): JSX.Element => {
     </Table>
   );
 };
+
+export const WithReactTable: Story = {
+  render: (): JSX.Element => <ReactTable />,
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+
+    // See https://storybook.js.org/docs/7.0/react/essentials/actions#automatically-matching-args to learn how to setup logging in the Actions panel
+    await userEvent.click(canvas.getByText("First Name"));
+
+    setTimeout(() => {
+      userEvent.click(canvas.getByText("Last Name"));
+    }, 2000);
+  },
```

Comment:
Is this doing anything yet? I looked locally, but not seeing any logging.

Reply:
Try to view this story in storybook. You will see that the columns change sorting after 2 seconds because it's actually 'playing' out a scene there.